### ISSUE TITLE
Update to Node 16

### ISSUE THREAD

**Issue by kuhnroyal** [andstor/file-existence-action#72]
For self-hosted runners on ARM64 (Mac M1) only Node 16 is supported. Would be great if the action could be updated.
Thanks!

**Comment by 7adityaraj** [comment]
Now we need to upgrade @andstor @khitrenovich @victorbnl @kuhnroyal 

```
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: andstor/file-existence-action, andstor/file-existence-action, andstor/file-existence-action, andstor/file-existence-action
```

**Comment by wojsmol** [comment]
cc @andstor

**Comment by andstor** [comment]
Thanks 🙏 
This should now be resolved in the new major release [v2.0.0](https://github.com/andstor/file-existence-action/releases/tag/v2.0.0).
